Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml

```diff
@@ -15,12 +15,12 @@
   - type: SolutionContainerManager
     solutions:
       food:
-        maxVol: 30
+        maxVol: 55 # flavored cakes have less room for spiking
         reagents:
         - ReagentId: Nutriment
-          Quantity: 20
+          Quantity: 40
         - ReagentId: Vitamin
-          Quantity: 5
+          Quantity: 10
```

Comment:
taking berries as an example,

source berries:
- nutriment: 5 * 3 = 15
- vitamin: 4 * 3 = 12
- total: 27u

source batter:
- 15u flour, 12u egg, 5u sugar, 5u milk: willing to call this as the other substantial half of this setup

checks out